lib/streamlit/__init__.py

```diff
@@ -580,6 +582,38 @@ def _transparent_write(*args):
     return args
 
 
+def _get_url():
```

Comment:
Based on item 1 from the root comment in this PR, this should be renamed `_get_query_string`, and it should return the query string.

It would also be great if the return value were a dict rather than a str. So if the query string is "?foo=123&bar=42&greetings=hello&greetings=hi" then the dict would be:

```
{
  foo: 123,
  bar: 42,
  greetings: ["hello", "hi"],
}
```

See [urllib.parse](https://docs.python.org/3/library/urllib.parse.html)